blueprints/examples/src/tests.rs

```diff
@@ -0,0 +1,136 @@
+use crate::eigen_context;
+use crate::eigen_context::ExampleTaskManager;
+use alloy_provider::Provider;
+use blueprint_test_utils::helpers::get_receipt;
+use blueprint_test_utils::incredible_squaring_helpers::start_default_anvil_testnet;
```

Comment:
Why is this apart of the incredible squaring helpers and not some `evm_test_utils`? Seems like the organization can be put in a properly named file. Incredible squaring stuff should be explicit to incredible squaring. Otherwise it's more general and reusable.